repositories.gradle

```diff
@@ -9,9 +13,6 @@ repositories {
             artifact()
         }
     }
-    maven {
-        url "https://cursemaven.com"
-    }
```

Comment:
Don't delete this, witchery is still fetched from there. Actually you could probably convert it to a deobf dep aswell and thus getting rid of the need for CCC

Reply:
deobf witchery can't handle the mixin it needs to do apparently:

```[13:24:25] [Server thread/ERROR] [FML]: Caught exception from witchery
java.lang.NoSuchMethodError: com.emoniph.witchery.brewing.potions.PotionParalysis.registerPotionAttributeModifier(Lnet/minecraft/entity/ai/attributes/IAttribute;Ljava/lang/String;DI)Lnet/minecraft/potion/Potion;
        at com.emoniph.witchery.brewing.potions.PotionParalysis.postContructInitialize(PotionParalysis.java:35) ~[PotionParalysis.class:?]
        at com.emoniph.witchery.brewing.potions.WitcheryPotions.register(WitcheryPotions.java:186) ~[WitcheryPotions.class:?]
        at com.emoniph.witchery.brewing.potions.WitcheryPotions.<init>(WitcheryPotions.java:62) ~[WitcheryPotions.class:?]```

All my normal tricks fail here, will read the curse version

Reply:
Maybe the mixins only contains the obfuscated name which works fine in non-dev but not in dev with an actually deobfuscated version of Witchery?

Reply:
Wait, no, there is not even Mixins involved in this case

Reply:
got a wacky runtime fix for now so runServer passes, but not sure what witchery wants to run in dev